src/ErrorBoundary.tsx

```diff
@@ -21,8 +22,11 @@ export default class ErrorBoundary extends Component<ErrorBoundaryProps, State>
         return {error};
     }
 
-    componentDidCatch() {
-        // do nothing
+    componentDidCatch(error: Error, info: ErrorInfo) {
+        const {componentDidCatch} = this.props;
+        if (componentDidCatch) {
```

Comment:
should I tell SuspenseError from normal one?

Reply:
Yes, for business developers `SuspenseError` should be invisible